### ISSUE TITLE
GetFullChat: "ERROR 400: CHAT_ID_INVALID"

### ISSUE THREAD

**Issue by alex2304** [LonamiWebs/Telethon#64]
Hello!

I'm trying to execute the following request:

```python
dialogs, entities = tg_client.get_dialogs(10)

    result = tg_client.invoke(
        GetFullChatRequest(
            entities[-1].id
        )
    )
```

Each time I get the same error: "ERROR 400: CHAT_ID_INVALID", if entities[-1] is a channel or a group(supergroup). 
When I've tried to pass into GetFullChatRequest() id of a user - I got almost the same error: "PEER_ID_INVALID". 

All the other methods I tried worked correctly, except GetChatsRequest() - error is the same.

Could you help, please? What am I doing wrong?

**Comment by Lonami** [comment]
Supergroups and channels are not chats, they're [channels](https://lonamiwebs.github.io/Telethon/methods/channels/get_full_channel.html).

**Comment by Lonami** [comment]
Did you manage to solve it?

**Comment by alex2304** [comment]
Thanks for response.

My goal was to get information about all the participants of a supergroup. I tried getFullChannel(), but it gave me only information about bots which are in the supergroup. Nothing about users.

Eventually, I decided to collect information about only such users that have been written something at least once. I retrieved all messages from the history, and gathered users in such way.

May be, when it will be necessary I'll try another method. Official documentation seems abandoned, and I've found a lot of new methods after discovering this fact :)
